kolibri/plugins/learn/assets/src/views/LibraryPage/UnPinnedDevices.vue

```diff
@@ -53,6 +62,10 @@
         required: false,
         default: 0,
       },
+      allDevices: {
```

Comment:
I wonder if this prop name could be changed just to `device`? As it seems from looking at the code this is just the information for a single device. `allDevices` suggests more than one to me!